### PR TITLE
Added ci.yml

### PR DESCRIPTION
**Pull request by IvanGrginovInf**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+            # Checkout repository
+            - name: Checkout
+              uses: actions/checkout@v2
+
+            # Setup PHP versions, run checks
+            - name: PHP setup
+              uses: shivammathur/setup-php@v2
+              with:
+                php-version: ${{ matrix.php }}
+                coverage: pcov
```
Comment (dingo-d) [inline .github/workflows/ci.yml:28]: You can remove this if you are not generating code coverage using GH actions

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                  ${{ runner.os }}-composer-
+
+            - name: Install composer packages
+              run: composer install --no-progress
+
+            - name: Check coding standards using PHPCS
+              run: composer standards:check -- --runtime-set testVersion ${{ matrix.php }}-
+
+    tests:
+        name: Codeception - Unit tests
```
Comment (dingo-d) [inline .github/workflows/ci.yml:50]: Let me know when you set up integration tests 😂

**Review by dingo-d — CHANGES_REQUESTED** [review]
One small change

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -46,6 +45,38 @@ jobs:
...
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+
+            - name: PHP setup
+              uses: shivammathur/setup-php@v2
+              with:
+                php-version: 7.2
+                coverage: pcov
```
Comment (dingo-d) [inline .github/workflows/ci.yml:59]: Coverage not needed

**Comment by IvanGrginovInf** [comment]
@dingo-d fixed all I could. I'm still getting the following error on almost all components, not sure why, seems like a bug:

`Expected 1 space before "!"; 0 foundPHPCS(WordPress.WhiteSpace.OperatorSpacing.NoSpaceBefore)`

For example on the following line (from `src/blocks/custom/input/input.php`):

```php
<?php ! empty( $input_id ) ? printf( 'id="%s"', esc_attr( $input_id ) ) : ''; ?>
```

🤔 

As for `phpstan` added, fixed some error but there's quite a few which I'll need to fix some other time unfortunately.

**Review by dingo-d — APPROVED** [review]
Approved. Hopefully, you'll be able to fix the issues 👀

**Comment by IvanGrginovInf** [comment]
Ofc @dingo-d right after I write tons of documentation I need to add for this plugin 😅

**Pull request merged**